mobile/src/containers/onboarding/BackupPhotosVideos.jsx

```diff
@@ -39,4 +41,4 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
   }
 })
 
-export default connect(mapStateToProps, mapDispatchToProps)(BackupPhotosVideos)
+export default connect(mapStateToProps, mapDispatchToProps)(translate()(BackupPhotosVideos))
```

Comment:
It's not `export default translate()(connect(mapStateToProps, mapDispatchToProps)(BackupPhotosVideos))`?